Sources/Swift/Tools/SentryViewPhotographer.swift

```diff
@@ -20,24 +20,24 @@ class DefaultViewRenderer: ViewRenderer {
 }
 
 @objcMembers
-class SentryViewPhotographer: NSObject, SentryViewScreenshotProvider {
+public class SentryViewPhotographer: NSObject, SentryViewScreenshotProvider {
```

Comment:
@philipphofmann will probably not agree with changing this to public, we should at least somehow mark this class as "internal use only"

Reply:
We should stop using hacky solutions and add a lot of "code smells" into the code.
If the user try to use an undocumented class, that is not part of the main product, let them try.

Reply:
I disagree with simply allowing users to use undocumented classes. If you allow it, people will use it, and then you might break them. I'm OK with making something public that shouldn't be use, if we prefix with something like InternalUseOnly_, so it's 100% clear. We already used that approach with `PrivateSentrySDKOnly` and never got complaints.